docs/sources/installation/debian.md

```diff
@@ -1,40 +1,45 @@
 +++
-title = "Installing on Debian / Ubuntu"
-description = "Install guide for Grafana"
+title = "Install on Debian/Ubuntu"
+description = "Install guide for Grafana on Debian or Ubuntu"
 keywords = ["grafana", "installation", "documentation"]
 type = "docs"
 aliases = ["/installation/installation/debian"]
 [menu.docs]
-name = "Installing on Ubuntu / Debian"
+name = "Install on Ubuntu/Debian"
 identifier = "debian"
 parent = "installation"
 weight = 200
 +++
 
-# Installing on Debian / Ubuntu
+# Install on Debian or Ubuntu
 
-Read [Upgrading Grafana]({{< relref "installation/upgrading.md" >}}) for tips and guidance on updating an existing
-installation.
+This page explains what the installation package does, how to install Grafana dependencies, download and install Grafana, and then get the service up and running on your system.
 
-## Download
+**Note on upgrading:** While the process for upgrading Grafana is very similar to installing Grafana, there are some key backup steps you should perform. Read [Upgrading Grafana]({{< relref "installation/upgrading.md" >}}) for tips and guidance on updating an existing installation.
 
-Go to the [download page](https://grafana.com/grafana/download?platform=linux) for the latest download
-links.
+## Package details
 
-```bash
-wget <debian package url>
-sudo apt-get install -y adduser libfontconfig1
-sudo dpkg -i grafana_<version>_amd64.deb
-```
+- Installs binary to `/usr/sbin/grafana-server`
+- Installs Init.d script to `/etc/init.d/grafana-server`
+- Creates default file (environment vars) to `/etc/default/grafana-server`
+- Installs configuration file to `/etc/grafana/grafana.ini`
+- Installs systemd service (if systemd is available) name `grafana-server.service`
+- The default configuration sets the log file at `/var/log/grafana/grafana.log`
+- The default configuration specifies an sqlite3 db at `/var/lib/grafana/grafana.db`
+- Installs HTML/JS/CSS and other Grafana files at `/usr/share/grafana`
```

Comment:
would like to have this below the install instructions as they are not relevant until after

Reply:
Done